musicdroid/src/androidTest/java/org/catrobat/musicdroid/pocketmusic/uitest/projectselection/ProjectSelectionActivityUiTest.java

```diff
@@ -200,4 +200,10 @@ public void testAutoRefresh() throws IOException, MidiException {
         solo.goBack();
         assertTrue(solo.searchText(fileName));
     }
+
```

Comment:
It is not good practice to hard code strings. This test fails f.e. in the german version of Musicdroid.